CHANGELOG.md

```diff
@@ -0,0 +1,159 @@
+# Changelog
+All notable changes to this project made by SYNAPTICON will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+##[Unreleased]
+### EtherCAT wrapper library
+#### Added
+- Enable reading/writing of string SDO values
+- Enable handling 64-bit integer values and setting the time of day
+- Check if the SDO request pointers are valid on upload/download
+- Add a function for bus rescanning
+- Add a function for preemptive bus rescanning
+- Enable preemptive slave vendor ID and product code fetching
+- Add a Synapticon changelog
+
+#### Fixed
+- Always null-terminate the string SDO values
+- Fix the domain registration during master initialization
+- Free the master domain in the proper place, when stopping the master
+- Fix segmentation fault on ecw_master_stop
+
+### EtherCAT library
+#### Added
+- Add a function to trigger bus rescanning
+
+#### Fixed
+- Suport Linux kernel 4.15.7
+- Fix: sncn_installer uses right sysconfig
+
+## [1.5.2-sncn-6] - 2018-10-23
+### EtherCAT wrapper library (v0.6)
+#### Added
+- Add version for libethercat_wrapper to distribution
```

Comment:
What was this? The ability get the version? In that case, just say "Add get_version()" (or whatever). Please no mention of the wrapper.